### PR TITLE
What is bigflow

### PR DESCRIPTION
**Pull request by beczkowb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,10 +23,27 @@
 1. Roadmap
 
 ## What is BigFlow?
-TODO
+
+BigFlow is a Python framework built on top of the [Cloud Composer](https://cloud.google.com/composer) ([Airflow](https://airflow.apache.org/)).
+It is made to simplify developing data processing pipelines on GCP.
+
+The main features are:
+
```
Comment (iwko) [inline README.md:31]: Scaffold?

File: README.md [hunk 2]
```diff
@@ -59,4 +76,5 @@ bigflow -h
 Read more about [BigFlow CLI](docs/cli.md).
 
 ## Help me
-TODO
\ No newline at end of file
+
+You can ask questions on our [Gitter channel](https://gitter.im/allegro/bigflow) or Stackoverflow.
```
Comment (iwko) [inline README.md:80]: Stackoverflow link?

File: README.md [hunk 3]
```diff
@@ -23,10 +23,27 @@
 1. Roadmap
 
 ## What is BigFlow?
-TODO
+
+BigFlow is a Python framework built on top of the [Cloud Composer](https://cloud.google.com/composer) ([Airflow](https://airflow.apache.org/)).
```
Comment (iwko) [inline README.md:27]: It can be used without composer too.

File: README.md [hunk 4]
```diff
@@ -23,10 +23,27 @@
...
+* [Support for the major data processing technologies](./docs/utils.md) — [Dataproc](https://cloud.google.com/dataproc),
+[Apache Beam](https://beam.apache.org/), [BigQuery](https://cloud.google.com/bigquery)
 
 ## Getting started
-TODO
+
+Start from [setting up a development environment](#installing-bigflow). 
+Next, go through [the BigFlow tutorial](./docs/tutorial.md). 
+
+Finally, you can dig into details about [project setup and build](./docs/project_setup_and_build.md), [deployment](./docs/deployment.md),
```
Comment (iwko) [inline README.md:45]: I think this paragraph is redundant. It duplicates `The main features` section.

File: README.md [hunk 5]
```diff
@@ -23,10 +23,27 @@
...
+[versioning](./docs/project_setup_and_build.md#project-versioning), [configuration](./docs/configuration.md)
+* [Unified project structure](./docs/project_setup_and_build.md#project-structure)
+* [Support for the major data processing technologies](./docs/utils.md) — [Dataproc](https://cloud.google.com/dataproc),
+[Apache Beam](https://beam.apache.org/), [BigQuery](https://cloud.google.com/bigquery)
 
 ## Getting started
-TODO
+
+Start from [setting up a development environment](#installing-bigflow). 
+Next, go through [the BigFlow tutorial](./docs/tutorial.md).
```
Comment (iwko) [inline README.md:43]: Next, go through the BigFlow [tutorial]

File: README.md [hunk 6]
```diff
@@ -23,10 +23,27 @@
...
+It is made to simplify developing data processing pipelines on GCP.
+
+The main features are:
+
+* [Dockerized jobs](./docs/project_setup_and_build.md#overview)
+* [Powerful CLI](./docs/cheatsheet.md)
+* [Automated build](./docs/project_setup_and_build.md#overview), [deployment](./docs/deployment.md), 
+[versioning](./docs/project_setup_and_build.md#project-versioning), [configuration](./docs/configuration.md)
+* [Unified project structure](./docs/project_setup_and_build.md#project-structure)
+* [Support for the major data processing technologies](./docs/utils.md) — [Dataproc](https://cloud.google.com/dataproc),
```
Comment (iwko) [inline README.md:37]: What will be in utils.md?

File: README.md [hunk 7]
```diff
@@ -23,10 +23,27 @@
...
+[Apache Beam](https://beam.apache.org/), [BigQuery](https://cloud.google.com/bigquery)
 
 ## Getting started
```
Comment (iwko) [inline README.md:40]: Maybe add link and info about scaffold?

File: README.md (hunk 3 above)
Comment (bartoszwalacik) [reply README.md:27]: właśnie, nie chcieliśmy się trochę odcinać od composera, że to tylko nasz scheduler, bo to zdanie brzmi jakbyśmy byli nadbudówką nad Composera

File: README.md [hunk 8]
```diff
@@ -23,10 +23,27 @@
 1. Roadmap
 
 ## What is BigFlow?
-TODO
+
+BigFlow is a Python framework built on top of the [Cloud Composer](https://cloud.google.com/composer) ([Airflow](https://airflow.apache.org/)).
+It is made to simplify developing data processing pipelines on GCP.
```
Comment (bartoszwalacik) [inline README.md:28]: [GCP](https://cloud.google.com/gcp/)

File: README.md (hunk 3 above)
Comment (iwko) [reply README.md:27]: @bartoszwalacik dokładnie tak jak piszesz zrozumiałem to zdanie

File: README.md [hunk 9]
```diff
@@ -23,10 +23,27 @@
...
+
+The main features are:
+
+* [Dockerized jobs](./docs/project_setup_and_build.md#overview)
+* [Powerful CLI](./docs/cheatsheet.md)
```
Comment (bartoszwalacik) [inline README.md:33]: `[Powerful CLI](./docs/cli.md)`

File: README.md [hunk 10]
```diff
@@ -23,10 +23,27 @@
...
 
 ## What is BigFlow?
-TODO
+
+BigFlow is a Python framework built on top of the [Cloud Composer](https://cloud.google.com/composer) ([Airflow](https://airflow.apache.org/)).
+It is made to simplify developing data processing pipelines on GCP.
+
+The main features are:
+
+* [Dockerized jobs](./docs/project_setup_and_build.md#overview)
```
Comment (bartoszwalacik) [inline README.md:32]: Dockerized jobs -> Dockerized deployment ?

File: README.md [hunk 11]
```diff
@@ -23,10 +23,27 @@
...
+* [Powerful CLI](./docs/cheatsheet.md)
+* [Automated build](./docs/project_setup_and_build.md#overview), [deployment](./docs/deployment.md),
```
Comment (bartoszwalacik) [inline README.md:34]: , [configuration] -> , **and** [configuration]

File: README.md (hunk 8 above)
Comment (bartoszwalacik) [inline README.md:28]: It is made to simplify **development and deployment** of data processing pipelines on [GCP](https://cloud.google.com/gcp/)

File: README.md [hunk 12]
```diff
@@ -23,10 +23,27 @@
...
+
+The main features are:
+
+* [Dockerized jobs](./docs/project_setup_and_build.md#overview)
+* [Powerful CLI](./docs/cheatsheet.md)
+* [Automated build](./docs/project_setup_and_build.md#overview), [deployment](./docs/deployment.md), 
+[versioning](./docs/project_setup_and_build.md#project-versioning), [configuration](./docs/configuration.md)
+* [Unified project structure](./docs/project_setup_and_build.md#project-structure)
+* [Support for the major data processing technologies](./docs/utils.md) — [Dataproc](https://cloud.google.com/dataproc),
+[Apache Beam](https://beam.apache.org/), [BigQuery](https://cloud.google.com/bigquery)
```
Comment (bartoszwalacik) [inline README.md:38]: a, b, **and** c

File: README.md (hunk 6 above)
Comment (bartoszwalacik) [inline README.md:37]: a new project scaffold ?

File: README.md [hunk 13]
```diff
@@ -23,10 +23,27 @@
...
+BigFlow is a Python framework built on top of the [Cloud Composer](https://cloud.google.com/composer) ([Airflow](https://airflow.apache.org/)).
+It is made to simplify developing data processing pipelines on GCP.
+
+The main features are:
+
+* [Dockerized jobs](./docs/project_setup_and_build.md#overview)
+* [Powerful CLI](./docs/cheatsheet.md)
+* [Automated build](./docs/project_setup_and_build.md#overview), [deployment](./docs/deployment.md), 
+[versioning](./docs/project_setup_and_build.md#project-versioning), [configuration](./docs/configuration.md)
+* [Unified project structure](./docs/project_setup_and_build.md#project-structure)
```
Comment (bartoszwalacik) [inline README.md:36]: Airflow DAG files generator

File: README.md (hunk 2 above)
Comment (beczkowb) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (beczkowb) [reply README.md]: check the current version

File: README.md (hunk 4 above)
Comment (beczkowb) [reply README.md]: removed

File: README.md (hunk 5 above)
Comment (beczkowb) [reply README.md:42]: done

File: README.md (hunk 6 above)
Comment (beczkowb) [reply README.md:36]:
```
1. Utils
    1. BigQuery
    1. Apache Beam
    1. Dataproc
    1. Job monitoring
```

File: README.md (hunk 7 above)
Comment (beczkowb) [reply README.md:40]: definetely

File: README.md (hunk 8 above)
Comment (beczkowb) [reply README.md]: done

File: README.md (hunk 9 above)
Comment (beczkowb) [reply README.md]: done

File: README.md (hunk 11 above)
Comment (beczkowb) [reply README.md:33]: done

File: README.md (hunk 12 above)
Comment (beczkowb) [reply README.md]: done

File: README.md (hunk 6 above)
Comment (beczkowb) [reply README.md:36]: done

File: README.md (hunk 13 above)
Comment (beczkowb) [reply README.md:35]: I think we might be the only people in the universe who do that. And it's hard to understand why, without any context. Contrary to, for example, automated build, deployment, and project structure. Those things are universal in tech.

File: README.md (hunk 8 above)
Comment (beczkowb) [reply README.md]: I have removed this sentence. I think that the main features speak for themselves.

File: README.md (hunk 10 above)
Comment (beczkowb) [reply README.md]: Maybe "Dockerized deployment environment"?

File: README.md (hunk 3 above)
Comment (iwko) [reply README.md]: 👍

File: README.md (hunk 6 above)
Comment (iwko) [reply README.md:36]: So maybe instead utils better name would be `processing`? And another section dedicated for `monitoring`?

**Review by iwko — APPROVED** [review]

File: README.md (hunk 6 above)
Comment (beczkowb) [reply README.md:36]: done

**Pull request merged**